exetera/core/fields.py

```diff
@@ -241,8 +269,12 @@ def complete(self):
 
 
 class MemoryFieldArray:
-
     def __init__(self, dtype):
+        """
+        Construct a memory based FieldArray which used as the wrapper of data in Field. The data is stored in numpy array.
```

Comment:
Construct a MemoryFieldArray instance. This class is an implementation detail, used to access data in non-indexed fields deriving from MemoryField. As such, instances of MemoryFieldArray should only be created by the Fields themselves, and not by users of ExeTera.